r/src/recordbatch.cpp

```diff
@@ -25,6 +25,8 @@
 #include <arrow/ipc/writer.h>
 #include <arrow/type.h>
 #include <arrow/util/key_value_metadata.h>
+#include <arrow/util/byte_size.h>
```

Comment:
Can you keep these includes alphabetically-ordered?

Reply:
Sure, will update this.